docs/paginator_configuration.md

```diff
@@ -15,6 +15,8 @@ knp_paginator:
         sort_field_name: sort          # sort field query parameter name; to disable sorting set this field to ~ (null)
         sort_direction_name: direction # sort direction query parameter name
         distinct: true                 # ensure distinct results, useful when ORM queries are using GROUP BY statements
+        page_out_of_range: ignore      # behavior if page number exceeds the last page number. Other options are 'fix' - replace page number with max page and 'throwException'
```

Comment:
this is way too long. We can remove explanation for "fix" option

Reply:
I shortened the description, maybe it will work now? I'd like to preserve information about what exactly it means here to "fix" if possible, as it isn't mentioned anywhere else in the bundle and could an be obscure option.